src/index.js

```diff
@@ -0,0 +1,79 @@
+import './index.css';
+import { deleteDataTodoRow } from './moduls/delete.js'; // eslint-disable-line
+import updateData from './moduls/update.js'; // eslint-disable-line
+import showData from './moduls/read.js'; // eslint-disable-line
+import completedTask from './moduls/completed.js'; // eslint-disable-line
+
+export const title = document.getElementById('title'), // eslint-disable-line
+  submit = document.getElementById('submit'),
+  updateDeleteItem = document.querySelector('.todo-tbody'),
+  clearDone = document.getElementById('clear-done');
+
+export let mood = 'creat'; // eslint-disable-line
+export let temp; // eslint-disable-line
+// let chekItem;
```

Comment:
Kindly note that leaving unused code in your code base can make your code really messy once it builds up.